force-app/main/default/permissionsets/HOT_Tolk_Formidler.permissionset-meta.xml

```diff
@@ -355,6 +355,11 @@
         <editable>false</editable>
         <field>HOT_Request__c.Duration__c</field>
         <readable>true</readable>
+    </fieldPermissions>
+        <fieldPermissions>
+        <editable>false</editable>
+        <field>HOT_Request__c.HOT_PersonContactId__c</field>
```

Comment:
Er dette noe formidler trenger å se på layouten, skal ikke denne bare brukes til å sette felter i bakgrunnen. Er vel Account formidler bør forholde seg til?

Reply:
Må ha tilgang til feltet for å kunne kjøre Oppdater felter til oppdrag-flyten